lib/BC/InstructionLifter.h

```diff
@@ -51,6 +51,10 @@
 #include "remill/BC/Util.h"
 #include "remill/OS/OS.h"
 
+namespace llvm{
+  class Type;
```

Comment:
The stylistically conformant formatting would be as follows:
```c++
namespace llvm {
class Type;
}  // namespace llvm
```

In general though, this shouldn't be needed because `Util.h` is included, and there is an identical forward declaration there.